src/embedded/utils.ts

```diff
@@ -28,12 +28,12 @@ export const handleElementClick = (
   appPackageName: string,
   errorCallback?: ErrorHandler
 ) => {
-  const clickedUrl = getClickedUrl(message?.elements?.defaultAction);
-  handleEmbeddedClick(clickedUrl);
+  const targetUrl = getClickedUrl(message?.elements?.defaultAction);
```

Comment:
Might be worth updating the fn too
```suggestion
  const targetUrl = getTargetUrl(message?.elements?.defaultAction);
```

Reply:
done!